hols/todo/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright (c) 2025 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dir=$1
+if [ $# -ne 1 ]; then
+  echo "Please provide directory"
+  exit 1
+fi
+
+if [ ! -d $dir ]; then
+  echo "$dir is not a directory"
+  exit 1
+fi
+
+DEBUG="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
+
+case $dir in
+  coherence)
+    cd coherence && java -Doracle.jdbc.provider.traceEventListener=open-telemetry-trace-event-listener-provider -Doracle.jdbc.provider.opentelemetry.sensitive-enabled=true -Dotel.export.name=todo.coherence -Dotel.sdk.disabled=false -Dotel.service.name=todo.coherence -Dotel.traces.exporter=otlp -Dotel.java.global-autoconfigure.enabled=true -Dcoherence.tracing.ratio=1 -Dotel.metrics.exporter=none -Dcoherence.hibernate.config=$HIBERNATE_CFG_XML -Dcoherence.wka=127.0.0.1 -Dcoherence.pof.enabled=true -Dcoherence.cluster=todo -jar target/helidon-labs-todo-coherence.jar
```

Comment:
Can these configuration be done via config instead ?
We should aim for `java -jar target/helidon-labs-todo-coherence.jar` and maybe add `-Dhelidon.config.profile=xxx`

Reply:
not sure I follow. can you please elaborate?

Reply:
Can you update the application so that these configuration can be expressed with config files instead of system properties ? If you use Helidon config then you can use config profiles to override the defaults.

You can also use meta-config to pickup optional config files in certain locations (E.g. from the current working directory ./etc/foo.yaml).

IIRC there is a Coherence mechanism to provide external configuration. You may be able to use Helidon SE config and propagate it to Coherence.

The goal is to reduce the number of command line arguments needed to start the application.
I.e. If the command is simply (`cd foo ; java -jar target/foo.jar` there is no need for a script because it's simple)

Reply:
yeah, can I do that in a subsequent PR?

Reply:
Yes, that's not a blocker for this PR.